src/control/server/ctl_storage_rpc.go

```diff
@@ -358,25 +373,35 @@ func (c *ControlService) doFormat(i *IOServerInstance, reformat bool, resp *ctlp
 // Send response containing multiple results of format operations on scm mounts
 // and nvme controllers.
 func (c *ControlService) StorageFormat(ctx context.Context, req *ctlpb.StorageFormatReq) (*ctlpb.StorageFormatResp, error) {
-	resp := new(ctlpb.StorageFormatResp)
-	resp.Mrets = proto.ScmMountResults{}
-	resp.Crets = proto.NvmeControllerResults{}
+	var resp *ctlpb.StorageFormatResp
+	respChan := make(chan *ctlpb.StorageFormatResp)
```

Comment:
Could make this a buffered channel for the number of expected responses. Minor improvement, not a big deal.

Reply:
done